.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Build sdist and wheel
         run: python setup.py sdist bdist_wheel
       - name: Publish to PyPI
-        uses: pypa/gh-action-pypi-publish@v1.3.1
+        uses: pypa/gh-action-pypi-publish@v1.4.1
```

Comment:
Isn't there a more generic tag like `v1`?

Reply:
no, not for that action, sadly.